webv2/session/session_handler.go

```diff
@@ -230,6 +230,14 @@ func SaveRemoteSession(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	sessionMetaData := GetSessionState().SessionMetadata
+	
+	sessionMetaData.DatabaseName = sm.DatabaseName
+	sessionMetaData.DatabaseType = sm.DatabaseType
+	sessionMetaData.SessionName = sm.SessionName
+
+	GetSessionState().SessionMetadata = sessionMetaData
```

Comment:
This looks weird 🤔
Why create a the temp variable `sessionMetaData`?
You can just do `GetSessionState().SessionMetadata.DatabaseName = sm.DatabaseName`